### PR TITLE
Rename `__init__.py` from v1 folder as modules.py

### PR DESCRIPTION
**Pull request by benflexcompute**

### CODE REVIEW

File: tests/test_current_flow360_version.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-from flow360.component.v1 import __version__
+from flow360.v1 import __version__
```
Comment (maciej-flexcompute) [inline tests/test_current_flow360_version.py:1]: why version is v1?

File: tests/test_environment.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
-from flow360.component.v1 import Env
+from flow360.v1 import Env
```
Comment (maciej-flexcompute) [inline tests/test_environment.py:1]: why env is v1?

File: tests/test_utils.py [hunk 3]
```diff
@@ -12,9 +12,9 @@
     shared_account_confirm_proceed,
     validate_type,
 )
-from flow360.component.v1 import Accounts
 from flow360.component.volume_mesh import VolumeMeshMeta
 from flow360.exceptions import Flow360TypeError, Flow360ValueError
+from flow360.v1 import Accounts
```
Comment (maciej-flexcompute) [inline tests/test_utils.py:17]: why Accounts is v1?

File: tests/test_current_flow360_version.py (hunk 1 above)
Comment (benflexcompute) [reply tests/test_current_flow360_version.py:1]: It will also be in V2

File: tests/test_environment.py (hunk 2 above)
Comment (benflexcompute) [reply tests/test_environment.py:1]: It will also be in B2

File: tests/test_utils.py (hunk 3 above)
Comment (benflexcompute) [reply tests/test_utils.py:17]: It will also be in V2

File: tests/test_current_flow360_version.py (hunk 1 above)
Comment (benflexcompute) [reply tests/test_current_flow360_version.py:1]: Change to import from absolute path

File: tests/test_environment.py (hunk 2 above)
Comment (benflexcompute) [reply tests/test_environment.py:1]: Change to import from absolute path

**Review by maciej-flexcompute — APPROVED** [review]

**Pull request merged**